{"name": "test_invalid_yaml_file", "status": "passed", "description": "Test validation with invalid YAML file.", "start": 1753819436330, "stop": 1753819436330, "uuid": "ddd330e3-6643-43a2-99cc-59b6aa803afb", "historyId": "695bdd8a27d9d604eec2049ce0e182df", "testCaseId": "695bdd8a27d9d604eec2049ce0e182df", "fullName": "tests.unit.core.test_validate_agent_resources_coverage.TestValidateAgentResources#test_invalid_yaml_file", "labels": [{"name": "parentSuite", "value": "tests.unit.core"}, {"name": "suite", "value": "test_validate_agent_resources_coverage"}, {"name": "subSuite", "value": "TestValidateAgentResources"}, {"name": "host", "value": "Yannick-Mac-Gillavry-werk.local"}, {"name": "thread", "value": "90219-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.unit.core.test_validate_agent_resources_coverage"}], "titlePath": ["tests", "unit", "core", "test_validate_agent_resources_coverage.py", "TestValidateAgentResources"]}